MiscVBAFunctions/Modules/MiscHasKey.bas

```diff
@@ -72,11 +72,12 @@ Public Function hasKey(Container, key As Variant) As Boolean
     On Error GoTo 0
     
     If ErrX = 0 Then ' No error trying to Access Key via .Item
-        If emptyFlag Then ' Item was Empty/non-existant
-            hasKey = False
-        Else
-            hasKey = True ' Item was not Empty
-        End If
+        'If emptyFlag Then ' Item was Empty/non-existant
```

Comment:
@BartJan142 , its fine to remove the commented code as we can see the diffs in the pull request